lensterxyz/lenster
# feat: add absolute timestamp tooltips

## What does this PR do?

Fixes #1269
For labels with relative time, add a tooltip with absolute timestamp

![Screenshot_2022-11-30_13-05-41](https://user-images.githubusercontent.com/667227/204792338-ff1764ba-edc3-43f6-89ea-24684a406018.png)


## Type of change


## How should this be tested?

Hover mouse cursor over a time label (for example "10 days ago") and check that an absolute date and time is shown, e.g. "10:31 am - 11/20/22"
